assets/model_monitoring/components/model_performance/model_performance_compute_metrics/spec.yaml

```diff
@@ -20,19 +20,14 @@ inputs:
       tabular-regression
     ]
     description: "Task type"
-  baseline_data:
-    type: mltable
-    mode: direct
   baseline_data_target_column:
     type: string
-    optional: true
     description: "Column name which contains ground truths in provided input for baseline_data. (Optional if we have one column name.)"
   production_data:
```

Comment:
so this is not the production data but the joined data right? can we add some description?

Reply:
Added descriptions.